### PR TITLE
🔍 Use comswasm-check for check contract

### PR DESCRIPTION
**Pull request by bdeneux**
Following @webmaster128 [comment](https://github.com/okp4/contracts/pull/12#issuecomment-1245452311) (thanks 😉), I've removed the experimental cw-check-contract crate and replaced by [cosmwasm-check](https://crates.io/crates/cosmwasm-check) crate.

### CODE REVIEW

File: Makefile.toml [hunk 1]
```diff
@@ -86,12 +86,12 @@ env = { RUSTFLAGS = "-C link-arg=-s" }
...
 [tasks.check_contracts]
 dependencies = ["wasm"]
-install_crate = { crate_name = "cw-check-contract" }
+install_crate = { crate_name = "cosmwasm-check" }
 script = '''
 for W in ./target/wasm32-unknown-unknown/release/*.wasm
 do
   echo -n "🔎 Checking `basename $W`... "
-  cw-check-contract $W
+  cosmwasm-check $W
```
Comment (webmaster128) [inline Makefile.toml:94]: You can even try `cosmwasm-check ./target/wasm32-unknown-unknown/release/*.wasm` directly now. But this is up to you of course.

Comment (bdeneux) [reply Makefile.toml]: Yes it's better. Thanks, the modification has been done ;)

**Review by amimart — APPROVED** [review]
👍

**Review by webmaster128 — APPROVED** [review]
Beautiful

**Review by ccamel — APPROVED** [review]

**Pull request merged**